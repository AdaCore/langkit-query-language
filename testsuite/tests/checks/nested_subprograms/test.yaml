driver: 'checker'
rule_name: Nested_Subprograms
input_sources: ['nested.adb']
