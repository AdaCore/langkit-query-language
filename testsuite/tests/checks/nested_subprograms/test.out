nested.adb:4:14: rule violation: subprogram declared in executable body
4 |    procedure Proc1;                    --  FLAG
  |              ^^^^^ 

nested.adb:6:14: rule violation: subprogram declared in executable body
6 |    procedure Proc2 is separate;        --  FLAG
  |              ^^^^^ 

